springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/types/channel/operation/message/Message.java

```diff
@@ -10,7 +10,8 @@
  */
 @Data
 @Builder
-@EqualsAndHashCode(of = {"name"})
```

Comment:
Same situation here as with the TreeSet, somehow this classes equality is only based on the name. I'm unsure if this alway holds true and it is very hard to find if there is an issue.  I'm not so sure what the original intention was and if this is really correct, but I would try not to make one object behave different than all others and rather code the special case where equality is only needed on the name field.